ddtrace/span.py

```diff
@@ -245,6 +245,10 @@ def set_exc_info(self, exc_type, exc_val, exc_tb):
         self.set_tag(errors.ERROR_TYPE, exc_type_str)
         self.set_tag(errors.ERROR_STACK, tb)
 
+    def set_pid(self, pid):
```

Comment:
I think we may not want to provide this public API, since at some point we're going to use a more standardized approach about what is the API of a `Span`. Said that, I think you can just move that code in the if branch when creating a root span.